public/docs/ts/latest/tutorial/toh-pt6.jade

```diff
@@ -34,13 +34,14 @@ block start-server-and-watch
     ## Keep the app transpiling and running
     Enter the following command in the terminal window:
 
-code-example(language="sh" class="code-shell").
-  npm start
+  code-example(language="sh" class="code-shell").
+    npm start
 
-:marked
-  This command runs the TypeScript compiler in "watch mode", recompiling automatically when the code changes.
-  The command simultaneously launches the app in a browser and refreshes the browser when the code changes.
+  :marked
+    This command runs the TypeScript compiler in "watch mode", recompiling automatically when the code changes.
+    The command simultaneously launches the app in a browser and refreshes the browser when the code changes.
```

Comment:
Lines 37-42 belong to the Jade `block` above it. This readjusts the indentation.